### ISSUE TITLE
Config.AE.h

### ISSUE THREAD

**Issue by kmsmi** [mosave/Beok2MQTT#1]
Где взять Config.AE.h ?

**Comment by kmsmi** [comment]
Я правильно понимаю что в этом файле только ssid и пароль для wifi? Или там еще что есть?

**Comment by mosave** [comment]
Да, в этом файле я держу только SSID и пароль для моей домашней wifi сети.
Нужно просто в свой конфиг добавить дефайн для  WIFI_SSID - и Config_AE.h станет не нужен :)

**Comment by kmsmi** [comment]
Ясно. Спасибо )

**Comment by kmsmi** [comment]
> Однако анализ данных, передаваемых по uart показал, что для управления MCU используется тот же протокол, что уже реализован в модуле Broadlink для MajorDoMo.

Не подскажешь где можно посмотреть отдельно про этот протокол для термостатов?
А то тут кода много, трудно сразу разобраться что для чего. Тем более если не знаешь что искать )

**Comment by mosave** [comment]
Только в этих исходниках и в исходниках модуля Broadlink для MajorDoMo:
https://github.com/nick7zmail/MajorDoMo-dev_broadlink
Но там будет упрощенная версия, поскольку инициализация MCU остается за скобками (ее реализует модуль broadlink)

**Comment by kmsmi** [comment]
Спасибо.
